aws/resource_aws_securityhub_account.go

```diff
@@ -0,0 +1,68 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/service/securityhub"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsSecurityHubAccount() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsSecurityHubAccountCreate,
+		Read:   resourceAwsSecurityHubAccountRead,
+		Delete: resourceAwsSecurityHubAccountDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{},
+	}
+}
+
+func resourceAwsSecurityHubAccountCreate(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).securityhubconn
+	log.Print("[DEBUG] Enabling Security Hub for account")
+
+	_, err := conn.EnableSecurityHub(&securityhub.EnableSecurityHubInput{})
+
+	if err != nil {
+		return fmt.Errorf("Error enabling Security Hub for account: %s", err)
+	}
+
+	d.SetId("securityhub-account")
```

Comment:
Would it make sense to make this the AWS account ID instead, so its usable as a reference in membership, etc?

```suggestion
	d.SetId(meta.(*AWSClient).accountid)
```